executor/aggregate_test.go

```diff
@@ -384,6 +384,9 @@ func (s *testSuite1) TestGroupConcatAggr(c *C) {
 	// issue #5411
 	tk := testkit.NewTestKit(c, s.store)
 	tk.MustExec("use test")
+	// https://github.com/pingcap/tidb/issues/6838 isn't implemented, so use this trick.
```

Comment:
We need more description here to explain why the issue matters.

Reply:
I think it's better revert this change, it can be done in 6838

Reply:
What do you mean by `this change`?

Reply:
Yes, group concat agg test change